dj_gui_api_server/DJConnector.py

```diff
@@ -310,6 +312,18 @@ def delete_tuple(jwt_payload: dict, schema_name: str, table_name: str,
         # All check pass thus proceed to delete
         tuple_to_delete.delete_quick()
 
+    @staticmethod
+    def get_table_object(schema_virtual_module, table_name: str):
+        # Split the table name by '.' for dealing with part tables
+        table_name_parts = table_name.split('.')
+        if len(table_name_parts) == 2:
+            table = getattr(getattr(schema_virtual_module,
+                                    table_name_parts[0]), table_name_parts[1])
+        else:
+            table = getattr(schema_virtual_module, table_name_parts[0])
+
+        return table
```

Comment:
This can be reduced down to 2 statements. 
For instance:
```suggestion
        table_name_parts = table_name.split('.')
        return (getattr(schema_virtual_module, table_name_parts[0])
                if len(table_name_parts) == 1
                else getattr(getattr(schema_virtual_module,
                                     table_name_parts[0]),
                             table_name_parts[1]))
```

Reply:
`# Split the table name by '.' for dealing with part tables
        table_name_parts = table_name.split('.')
        if len(table_name_parts) == 2:
            return getattr(getattr(schema_virtual_module,
                                    table_name_parts[0]), table_name_parts[1])
        else:
            return getattr(schema_virtual_module, table_name_parts[0])`

Reply:
Can we just go with this ^. I typically don't like those if and else 1 line statement since it can get clutter at times.